### ISSUE TITLE
Problem with line breaks

### ISSUE THREAD

**Issue by leogermani** [gavinr/github-csv-tools#6]
Hi,

Trying to use this module here and get the following error when I call the command via CLI

```
env: node\r: No such file or directory
```
Ive found another module that had the same issue but I could not solve it using the methods described there: https://github.com/nwjs-community/nw-builder/issues/75

I never coded a npm module before so I dont know where to start to look to fix it...

Hope you have time to look into it

thanks

**Comment by leogermani** [comment]
Im on a linux machive (ubuntu) by the way

**Comment by leogermani** [comment]
I also tried what is described in https://github.com/gavinr/github-csv-tools/issues/5

both on `/usr/local/bin/githubCsvTools` and `/usr/local/lib/node_modules/github-csv-tools/index.js`

**Comment by gavinr** [comment]
Thanks for the report @leogermani! Will try to take a look shortly.

**Comment by slamorte** [comment]
Ditto here on MacOS 10.12.05. 

`env: node\r: No such file or directory`

A workaround is to call it from node:

`env node github-csv-tools`

However I get errors doing this, probably due to an outdated node, so I'm updating node now (which is another set a issues and errors).

**Comment by slamorte** [comment]
fixed with:

`brew install dos2unix`

`sudo dos2unix /usr/local/lib/node_modules/github-csv-tools/index.js`

now `githubCsvTools --help` works correctly.

**Comment by tflogic** [comment]
on linux(ubuntu)
```
sudo apt install dos2unix
sudo dos2unix /usr/local/lib/node_modules/github-csv-tools/index.js

```

**Comment by gavinr** [comment]
@tflogic @slamorte @leogermani I have re-published v0.3.0 with unix-style line breaks instead of windows - can you please remove/reinstall and let me know if that helps?

**Comment by slamorte** [comment]
Thanks Gavinr! This script has been an awesome resource for me. 

Probably can't test it for a few weeks though, because holidays. =)
